diff --git a/node_modules/react-native-paper/src/components/.DS_Store b/node_modules/react-native-paper/src/components/.DS_Store
new file mode 100644
index 0000000..2803966
Binary files /dev/null and b/node_modules/react-native-paper/src/components/.DS_Store differ
diff --git a/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx b/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx
index 815da67..38a08cf 100644
--- a/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx
+++ b/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx
@@ -763,12 +763,7 @@ const BottomNavigationBar = <Route extends BaseRoute>({
               children: (
                 <View
                   pointerEvents="none"
-                  style={
-                    isV3 &&
-                    (labeled
-                      ? styles.v3TouchableContainer
-                      : styles.v3NoLabelContainer)
-                  }
+                  style={styles.v3NoLabelContainer}
                 >
                   <Animated.View
                     style={[
@@ -878,7 +873,6 @@ const BottomNavigationBar = <Route extends BaseRoute>({
                             maxFontSizeMultiplier={labelMaxFontSizeMultiplier}
                             variant="labelMedium"
                             style={[
-                              styles.label,
                               {
                                 color: activeLabelColor,
                                 ...font,
@@ -908,7 +902,6 @@ const BottomNavigationBar = <Route extends BaseRoute>({
                               variant="labelMedium"
                               selectable={false}
                               style={[
-                                styles.label,
                                 {
                                   color: inactiveLabelColor,
                                   ...font,
@@ -950,17 +943,12 @@ const styles = StyleSheet.create({
   },
   items: {
     flexDirection: 'row',
-    ...(Platform.OS === 'web'
-      ? {
-          width: '100%',
-        }
-      : null),
+    
   },
   item: {
     flex: 1,
     // Top padding is 6 and bottom padding is 10
     // The extra 4dp bottom padding is offset by label's height
-    paddingVertical: 6,
   },
   v3Item: {
     paddingVertical: 0,
@@ -969,17 +957,14 @@ const styles = StyleSheet.create({
     position: 'absolute',
   },
   iconContainer: {
-    height: 24,
-    width: 24,
-    marginTop: 2,
-    marginHorizontal: 12,
+    
+    
     alignSelf: 'center',
   },
   v3IconContainer: {
     height: 32,
     width: 32,
-    marginBottom: 4,
-    marginTop: 0,
+    
     justifyContent: 'center',
   },
   iconWrapper: {
@@ -997,28 +982,14 @@ const styles = StyleSheet.create({
     ...StyleSheet.absoluteFillObject,
   },
   // eslint-disable-next-line react-native/no-color-literals
-  label: {
-    fontSize: 12,
-    height: BAR_HEIGHT,
-    textAlign: 'center',
-    backgroundColor: 'transparent',
-    ...(Platform.OS === 'web'
-      ? {
-          whiteSpace: 'nowrap',
-          alignSelf: 'center',
-        }
-      : null),
-  },
+  
   badgeContainer: {
     position: 'absolute',
     left: 0,
   },
-  v3TouchableContainer: {
-    paddingTop: 12,
-    paddingBottom: 16,
-  },
+  
   v3NoLabelContainer: {
-    height: 80,
+    height: 40,
     justifyContent: 'center',
     alignItems: 'center',
   },
